test/airhorn.test.ts

```diff
@@ -86,15 +86,29 @@ test('Airhorn - Send SMTP', async () => {
 });
 
 test('Airhorn - Send Mobile Push', async () => {
```

Comment:
Can we remove this test as it has sensitive information. Anyway to do it as a mock?

Reply:
We don't have sensitive information, is a sample data.